docs/framework/deployment/deployment-guide-for-developers.md

```diff
@@ -353,11 +353,11 @@ The following table lists options that you can include when you chain the .NET F
 |Option|Description|
 |------------|-----------------|
 |**/CEIPConsent**|Overwrites the default behavior and sends anonymous feedback to Microsoft to improve future deployment experiences. This option can be used only if the setup program prompts for consent and if the user grants permission to send anonymous feedback to Microsoft.|
-|**/chainingpackage** `packageName`|Specifies the name of the executable that is doing the chaining. This information is sent to Microsoft as anonymous feedback to help improve future deployment experiences.<br /><br /> If the package name includes spaces, use double quotation marks as delimiters; for example: **/chainingpackage "Lucerne Publishing"**. For an example of a chaining package, see [Getting Progress Information from an Installation Package](https://go.microsoft.com/fwlink/?LinkId=181926) in the MSDN Library.|
+|**/chainingpackage** `packageName`|Specifies the name of the executable that is doing the chaining. This information is sent to Microsoft as anonymous feedback to help improve future deployment experiences.<br /><br /> If the package name includes spaces, use double quotation marks as delimiters; for example: **/chainingpackage "Lucerne Publishing"**. For an example of a chaining package, see [Getting Progress Information from an Installation Package](https://docs.microsoft.com/previous-versions/cc825975(v=vs.100)).|
 |**/LCID**  `LCID`<br /><br /> where `LCID` specifies a locale identifier (see [supported languages](#supported-languages))|Installs the language pack specified by `LCID` and forces the displayed UI to be shown in that language, unless quiet mode is set.<br /><br /> For the web installer, this option chain-installs the language package from the web. **Note:**  Use this option only with the web installer.|
 |**/log** `file` &#124; `folder`|Specifies the location of the log file. The default is the temporary folder for the process, and the default file name is based on the package. If the file extension is .txt, a text log is produced. If you specify any other extension or no extension, an HTML log is created.|
 |**/msioptions**|Specifies options to be passed for .msi and .msp items; for example: `/msioptions "PROPERTY1='Value'"`.|
-|**/norestart**|Prevents the setup program from rebooting automatically. If you use this option, the chaining app has to capture the return code and handle rebooting (see [Getting Progress Information from an Installation Package](https://go.microsoft.com/fwlink/?LinkId=179606) in the MSDN Library).|
+|**/norestart**|Prevents the setup program from rebooting automatically. If you use this option, the chaining app has to capture the return code and handle rebooting (see [Getting Progress Information from an Installation Package](https://docs.microsoft.com/previous-versions/cc825975(v=vs.100))).|
```

Comment:
One too many `)` in the url. 😄 

```suggestion
|**/norestart**|Prevents the setup program from rebooting automatically. If you use this option, the chaining app has to capture the return code and handle rebooting (see [Getting Progress Information from an Installation Package](https://docs.microsoft.com/previous-versions/cc825975(v=vs.100)).|
```

Reply:
The link is actually inside parentheses so I think that’s correct

Reply:
![0D1E01A9-2049-45BE-8AF8-1F63C56CCFC0](https://user-images.githubusercontent.com/12971179/74058816-b09c1900-499b-11ea-9b10-bcfe481fe553.jpeg)